### PR TITLE
199 smoke test for display points gherkin

### LINKED ISSUE #199: Write Smoke test for display-points Gherkin

### PR DESCRIPTION
**Pull request by qpb948**
Closes #199 

# What was the issue
Needed smoke test for gherkins.

# What was done and why
Created a smoke test for displaying data points. Used test hooks to log to a file and then search the log for the correct hooks.

# How it was tested
I walked through the system and tested the test hooks go to the log, and ensure the output is correct. And the test cases pass.

### CODE REVIEW

**Review by Subzeero — CHANGES_REQUESTED** [review]
It looks like Git messed up some thing in the PointSelectionContext

**Review by MitchWag01 — COMMENTED** [review]
Is there any way for us to verify that specified data has actually shown up? just looking through the gherkin it says the data point displays "specified" data? could we possibly check that its showing the correct information?

File: my-webxr-app/src/contexts/PointSelectionContext.tsx
[the remark this answers is not in the document]
Comment (qpb948) [reply my-webxr-app/src/contexts/PointSelectionContext.tsx:1]: I believe I replaced it correctly, if you could take a look to double check. thanks

**Review by Subzeero — APPROVED** [review]
Looks good 👍

**Review by LongQuanNguyen — APPROVED** [review]
Good works!

**Pull request merged**